shell/platform/windows/accessibility_bridge_windows.cc

```diff
@@ -41,10 +41,16 @@ void AccessibilityBridgeWindows::OnAccessibilityEvent(
       DispatchWinAccessibilityEvent(win_delegate,
                                     ax::mojom::Event::kChildrenChanged);
       break;
-    case ui::AXEventGenerator::Event::DOCUMENT_SELECTION_CHANGED:
+    case ui::AXEventGenerator::Event::DOCUMENT_SELECTION_CHANGED: {
```

Comment:
Could we add a comment similar to Chromium's comment? https://source.chromium.org/chromium/chromium/src/+/main:content/browser/accessibility/browser_accessibility_manager_win.cc;l=230-234?q=case%20ui::AXEventGenerator::Event::DOCUMENT_SELECTION_CHANGED&ss=chromium

Reply:
Does this part of the comment make sense for us too?

> This is because the focus is the only endpoint that can move, and because the caret (if present) is at the focus.

Reply:
I do not think so, as we do not have the carat as its own node.